tests/admin_views/models.py

```diff
@@ -330,6 +330,10 @@ class Meta:
         ordering = ('release_date',)  # overridden in PodcastAdmin
 
 
+class PodcastInfo(models.Model):
```

Comment:
Did you try reusing an existing model? We try to avoid adding more models since each one increases the duration of the test suite.